src/app/admin/admin-products/admin-products.component.ts

```diff
@@ -1,7 +1,14 @@
+import { ProductService } from './../../product.service';
 import { Component, OnInit } from '@angular/core';
 
 @Component({
   selector: 'app-admin-products',
   templateUrl: './admin-products.component.html'
 })
-export class AdminProductsComponent {}
+export class AdminProductsComponent {
+  products$;
+
+  constructor(private productService: ProductService) {
+    this.products$ = this.productService.getAll();
```

Comment:
Probably should change products$ productsObservable

Reply:
Good catch.  Done.